overlord/snapstate/snapmgr.go

```diff
@@ -716,6 +716,7 @@ func (m *SnapManager) doLinkSnap(t *state.Task, _ *tomb.Tomb) error {
 	if err != nil {
 		return err
 	}
+	t.Logf("snap %q at rev %s linked", newInfo.Name(), newInfo.Revision)
```

Comment:
This shows up in the changes UI too. Suggest something like:

`"snap %q at revision %d made available to the system."`